src/main/java/com/iotiq/application/service/CustomerService.java

```diff
@@ -124,12 +124,13 @@ public void createCustomer(User currentUser) {
     }
 
     private List<OrderDto> getLastTwoOrders(List<Order> orderList) {
+
         if (CollectionUtils.isEmpty(orderList)) {
             return Collections.emptyList();
         }
         return orderList.stream()
-                .map(order -> order == null ? null : ModelMapperUtil.map(order, OrderDto.class))
                 .filter(orderDto -> orderDto != null && orderDto.isVisible())
```

Comment:
not orderDto, order. You can apply limit before mapping.

Reply:
i cant apply limit before sorting, it affects output